tools/accuracy_checker/accuracy_checker/postprocessor/nms.py

```diff
@@ -131,6 +131,29 @@ def nms(x1, y1, x2, y2, scores, thresh, include_boundaries=True, keep_top_k=None
 
         return keep
 
+class BatchedNMS(NMS):
+    __provider__ = 'batched_nms'
```

Comment:
it is not batched nms, it is class aware nms

Reply:
renamed